Jenkinsfile

```diff
@@ -62,7 +62,7 @@ pipeline {
                 branch 'master'
             }
             steps {
-                sh './mvnw clean deploy -Papache-release -Dgpg.skip=true --batch-mode -pl -:brave-itests'
+                sh './mvnw deploy -Papache-release -Dgpg.skip=true --batch-mode -pl -:brave-itests'
```

Comment:
I should have been more clear in my suggestion. This is still using the `deploy` *phase* which will run all phases before it (which happen to be all the rest). I think if we use the Maven Deploy Plugin's `deploy` *goal* directly instead we can just do the deploy step without running all the [Maven lifecycle phases](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html) up to it (which were run in the previous stage). To be clear, I think this would be most direct and succinct:

```
sh './mvnw deploy:deploy -Papache-release -Dgpg.skip=true --batch-mode -pl -:brave-itests'
```

Reply:
Ah, thank you for the explanation and sorry for my ignorance on all things Maven :D This makes sense, especially since with your way we won't quietly build any missing dependencies, but instead fail loudly. Which is what we want here.

Reply:
https://github.com/apache/incubator-zipkin-brave-karaf/pull/23